source/tests/SheeshWalkerTest.kt

```diff
@@ -0,0 +1,13 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertEquals
+
+class SheeshWalkerTest {
+    @Test
+    fun `Should return string with exact same speed and name of walker specified in SheeshWalker class`() {
+        val speed = 20
+
+        val actual = SheeshWalker(speed)
```

Comment:
Actual must be final. But u'r use in the assert actual.walk
It's bad practice
Move calling method to actual initialisation